### PR TITLE
Directoried Git Notes

### PR DESCRIPTION
**Pull request by santi-h**

### CODE REVIEW

**Comment by jeet4320** [comment]
we should close this :)

File: test/notes.py [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+  def testFileToOpenWithIdentifier(self):
+    dirpath = join(files_dirpath(), 'notes', 'test-2')
+    notes = Notes(None, None)
+    notes._notes_directory = MagicMock(return_value = dirpath)
+    test_cases = [
+      [None,          None],
+      ['',            None],
+      ['5',           None],
+      ['-',           None],
+      ['0',           join(dirpath, '0-some-file.txt')],
```
Comment (whatisdot) [inline test/notes.py]: Can't you create these files in the file system before the test and them destroy them during cleanup?

Comment (santi-h) [reply test/notes.py]: @whatisdot this is not ready and will probably not merge it since I'm the only one using it, so don't worry about reviewing

Comment (santi-h) [reply test/notes.py]: I'll close it

**Pull request closed without merging**